src/components/OrgListCard/OrgListCard.tsx

```diff
@@ -112,8 +112,12 @@ function orgListCard(props: InterfaceOrgListCardProps): JSX.Element {
             )}
             {/* Display the number of admins and members */}
             <h6 className={styles.orgadmin}>
-              {tCommon('admins')}: <span>{admins.length}</span> &nbsp; &nbsp;
-              &nbsp; {tCommon('members')}: <span>{members.length}</span>
+              <div>
+                {tCommon('admins')}: <span>{admins.length}</span>
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`, with exceptions for package based CSS.
1. Delete the CSS file it currently uses.
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296